wls-eai-service/src/main/java/de/muenchen/oss/wahllokalsystem/eaiservice/rest/wahlvorschlag/dto/KandidatDTO.java

```diff
@@ -0,0 +1,12 @@
+package de.muenchen.oss.wahllokalsystem.eaiservice.rest.wahlvorschlag.dto;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.validation.constraints.NotNull;
+
+public record KandidatDTO(@NotNull String identifikator,
+                          @NotNull String name,
+                          @Schema(requiredMode = Schema.RequiredMode.REQUIRED) long listenposition,
+                          boolean direktkandidat,
```

Comment:
```suggestion
                         @NotNull boolean direktkandidat,
```
Im Altsystem ist @NotNull

Reply:
fixed mit 5d0fdade249a651e9e3405c4881bf8e2358d1ff5